packages/Config/RectorConfig.php

```diff
@@ -50,7 +50,7 @@ public function disableParallel(): void
         $parameters->set(Option::PARALLEL, false);
     }
 
-    public function parallel(int $seconds = 120, int $maxNumberOfProcess = 16, int $jobSize = 20): void
+    public function parallel(int $seconds = 120, int $maxNumberOfProcess = 0, int $jobSize = 20): void
```

Comment:
since the parameter `$maxNumberOfProcess` has in-depth semantics I think we need a phpdoc explaining the different use-cases one can do with it

Reply:
very good idea! phpdoc added, ping @TomasVotruba

Reply:
@TomasVotruba thanks for the feedback! what would be your ideal API to configure preserving CPU cores?

To me it feels logical, I read "$maxNumberOfProcess = -1" as "minus one CPU core from all available CPU cores", or in a more lengthy way "perserve one CPU core amoung all available CPU cores"
Any idea how to make this clearer?
It would be too much and confusing to add a new parameter for this preserve configuration IMO.

Goal is to provide a persistent configuration among a dev team, where devs don't have the same number of CPU cores on their laptop.